src/main/webapp/Index.html

```diff
@@ -1,647 +1,580 @@
+                        <td>
+                            <dl>
+                                <dt class="text-primary">Samsung TV</dt>
+                                <dd class="text-black-50">Samsung 32" 1080p 60Hz LED ....</dd>
+                            </dl>
+                        </td>
+                        <td align="center" style="vertical-align: middle;">
+                        	<span class="badge bg-warning">$1800</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td align="center" style="vertical-align: middle;">
+                        	<i class="fa-sharp fa-regular fa-image fa-2xl"></i>
+                        </td>
+                        <td>
+                            <dl>
+                                <dt class="text-primary">Bicycle</dt>
+                                <dd class="text-black-50">26" Mangoos Dolomite Men's...</dd>
+                            </dl>
+                        </td>
+                        <td align="center" style="vertical-align: middle;">
+                        	<span class="badge bg-primary">$700</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td align="center" style="vertical-align: middle;">
+                        	<i class="fa-sharp fa-regular fa-image fa-2xl"></i>
+                        </td>
+                        <td>
+                            <dl>
+                                <dt class="text-primary">Xbox One</dt>
+                                <dd class="text-black-50">Xbox One Console Blundel with......</dd>
+                            </dl>
+                        </td>
+                        <td align="center" style="vertical-align: middle;">
+                        	<span class="badge bg-danger">$1800</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td align="center" style="vertical-align: middle;">
+                       		<i class="fa-sharp fa-regular fa-image fa-2xl"></i>
+                        </td>
+                        <td>
+                            <dl>
+                                <dt class="text-primary">PlayStation 4</dt>
+                                <dd class="text-black-50">PlayStation 4 500GB Console(PS4)</dd>
+                            </dl>
+                        </td>
+                        <td align="center" style="vertical-align: middle;">
+                        	<span class="badge bg-success">$1800</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td colspan="3" align="center"><a href="#">View All Products</a></td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+    </div> 
+    
+    <!-- Side-navigation script -->
+    <script>
+        var menu_btn = document.querySelector("#menu-btn")
```

Comment:
Why we need script to open and close side bar. Just remove it and use bootstrap insted.